chapter_22_fixtures_and_wait_decorator.asciidoc

```diff
@@ -100,7 +100,7 @@ against the database on the staging server too.
 .Django Sessions: How a User's Cookies Tell the Server She Is Authenticated
 **********************************************************************
 
-_Being an attempt to explain sessions, cookies, and authentication in Django._
+_An attempt to explain sessions, cookies, and authentication in Django._
```

Comment:
aw i like the `being xyz` form for chapter subtitles, or whatever you call them.  it's old-fashioned.